src/templates/ui-grid/uiGridHeaderCell.html

```diff
@@ -10,35 +10,41 @@
     class="ui-grid-cell-contents ui-grid-header-cell-primary-focus"
     col-index="renderIndex"
     title="TOOLTIP">
-    <span
+    <div
```

Comment:
What is the extra div for?

Reply:
To enable the ellipses text overflow, the inner div should have overflow set to hidden. At the same time, the sort information should not be hidden, although it is part of the overflowing content. To solve this conflict I introduced the outer div.

Reply:
thanks for the explanation.